src/LockstakeEngine.sol

```diff
@@ -106,18 +107,18 @@ contract LockstakeEngine is Multicall {
     event AddFarm(address farm);
     event DelFarm(address farm);
     event Open(address indexed owner, uint256 indexed index, address urn);
-    event Hope(address indexed urn, address indexed usr);
-    event Nope(address indexed urn, address indexed usr);
-    event SelectVoteDelegate(address indexed urn, address indexed voteDelegate);
-    event SelectFarm(address indexed urn, address farm, uint16 ref);
-    event Lock(address indexed urn, uint256 wad, uint16 ref);
-    event LockNgt(address indexed urn, uint256 ngtWad, uint16 ref);
-    event Free(address indexed urn, address indexed to, uint256 wad, uint256 freed);
-    event FreeNgt(address indexed urn, address indexed to, uint256 ngtWad, uint256 ngtFreed);
-    event FreeNoFee(address indexed urn, address indexed to, uint256 wad);
-    event Draw(address indexed urn, address indexed to, uint256 wad);
-    event Wipe(address indexed urn, uint256 wad);
-    event GetReward(address indexed urn, address indexed farm, address indexed to, uint256 amt);
+    event Hope(address indexed owner, uint256 indexed index, address indexed usr);
+    event Nope(address indexed owner, uint256 indexed index, address indexed usr);
+    event SelectVoteDelegate(address indexed owner, uint256 indexed index, address indexed voteDelegate);
+    event SelectFarm(address indexed owner, uint256 indexed index, address indexed farm, uint16 ref);
```

Comment:
Note `indexed` was added to the `farm` value to keep it consistent with `voteDelegate`.